pom.xml

```diff
@@ -354,6 +354,10 @@
               <argument>--mount</argument>
               <argument>type=tmpfs,target=/opt/cryostat.d/truststore.d</argument>
               <argument>--env</argument>
+              <argument>CRYOSTAT_MAX_TARGET_CONNECTIONS=1</argument>
```

Comment:
I'm guessing the previous default value of unlimited target connections was unnecessary since we typically stick to 1 target when testing etc. anyways?

Reply:
Well, there is the `InterleavedExternalTargetRequestsIT`, which does have multiple targets and intentionally connects to them in a varying order. That test was written pretty much specifically to exercise the TargetConnectionManager and ensure that the caching behaviour works as expected and you don't end up with requests going to the wrong target etc. However, the number of concurrent connections open is really a performance optimization, and from an API perspective it doesn't matter, so the behaviour should be the same regardless of the cache size or number of targets.

I set it to 1 here again mostly just to ensure things don't blow up, but it might make sense to either set it to 0 for immediate eviction (highest test suite stress) or -1/unset for unbounded cache (also could cause stress if another test is written which has very many targets, but in general this would result in the fastest test suite execution time).

Reply:
Oh, that clears it up then, I didn't realize the connection concurrency was a performance optimization; thought it was a strict bound on the number of targets able to be connected to.

Reply:
Yea, I realized the name was misleading, so I pushed a commit renaming the new variables in a way that (I hope) carries my intent better.

Reply:
Nice, the new variable names are definitely more clear.